src/routeObject.js

```diff
@@ -1,6 +1,67 @@
 import React from 'react';
 import {Route, IndexRoute} from 'react-router';
+
 import App from './pages/App';
+import NotFound from './pages/PageNotFound';
+import store from './store';
+
+const lessons = store.getState().lessons;
+const courses = store.getState().context['courseContext'].keys();
+
+const getCleanCoursePaths = (courseArray) => {
```

Comment:
Siden du likevel skal gjøre de to andre endringene, så foreslår jeg:
```
const getCleanCoursePaths = (courseArray) =>
  courseArray.map((course) => course.slice(1).replace(/\/index.md/i, ''));

const getLessonPaths = (lessonObject) =>
  Object.keys(lessonObject).map((key) => lessonObject[key]['path']);
```

...men hvis vi først kommer så langt, så kunne man kanskje likegodt droppe funksjonene:
```
const lessonArray = Object.keys(lessons).map((key) => lessons[key]['path']);
const courseArray = courses.map((course) => course.slice(1).replace(/\/index.md/i, ''));
```